### DISCUSSION TITLE
Is that possible to toggle the visibility of the spawned vehicles or walkers?

### DISCUSSION THREAD

**Discussion by LieYuan1993** [carla-simulator/carla#5420] [Q&A]
Carla provides an API to toggle the environment objects' visibility. Out of curiosity, I want to know if there is a way to change the code and toggle the visibility of the spawned actors. Could someone let me know, please?

![EnvironmentObjects](https://user-images.githubusercontent.com/96498544/167126987-d28fa743-6a1d-47f1-8f3d-f12f761535a8.png)

**Comment by ffunatsu** [comment]
Currently impossible using PythonAPI (https://github.com/carla-simulator/carla/issues/4261).
